tests/client/client_test.go

```diff
@@ -283,31 +292,36 @@ func (s *clientTestSuite) TestLocalTSO(c *C) {
 	c.Assert(err, IsNil)
 
 	wg := sync.WaitGroup{}
-	for _, dcLocation := range dcLocationConfig {
-		wg.Add(1)
-		go func(dc string) {
-			defer wg.Done()
-			var err error
-			var p1, l1 int64
-			testutil.WaitUntil(c, func(c *C) bool {
-				p1, l1, err = cli.GetLocalTS(context.TODO(), dc)
-				if err == nil {
-					return true
-				}
-				c.Log(err)
-				return false
-			})
-			time.Sleep(10 * time.Millisecond)
-			testutil.WaitUntil(c, func(c *C) bool {
-				p2, l2, err := cli.GetLocalTS(context.TODO(), dc)
-				if err != nil {
+	for i := 0; i < 100; i++ {
+		for _, dcLocation := range dcLocationConfig {
+			wg.Add(1)
+			go func(dc string) {
+				defer wg.Done()
+				var ts1, ts2 uint64
+				testutil.WaitUntil(c, func(c *C) bool {
+					p1, l1, err := cli.GetLocalTS(context.TODO(), dc)
+					if err == nil {
+						ts1 = tsoutil.ComposeTS(p1, l1)
+						return true
+					}
+					c.Log(err)
+					return false
+				})
+				c.Assert(cluster.CheckTSOUnique(ts1), IsTrue)
+				time.Sleep(10 * time.Millisecond)
+				testutil.WaitUntil(c, func(c *C) bool {
```

Comment:
Similar logical in the same function is ugly, how about using a temporary to store the previous `TS`, which may similar to 
```

    prevTS := uint64(0)
	for i := 0; i < 5; i++ {
		var curTS uint64
		testutil.WaitUntil(c, func(c *C) bool {
			....
				curTS = tsoutil.ComposeTS(p1, l1)
			...
		})
		c.Assert(cluster.CheckTSOUnique(curTS), IsTrue)
		c.Assert(prevTS, Less, curTS)
		prevTS = curTS
		time.Sleep(10 * time.Millisecond)
	}
```